data/data/bootstrap/bootstrap-in-place/files/opt/openshift/bootstrap-in-place/bootstrap-in-place-post-reboot.sh

```diff
@@ -25,12 +25,15 @@ function signal_bootstrap_complete {
 }
 
 function restart_kubelet {
-  echo "Restarting kubelet"
-  until [ "$(oc get pod -n openshift-kube-apiserver-operator --selector='app=kube-apiserver-operator' -o jsonpath='{.items[0].status.conditions[?(@.type=="Ready")].status}' | grep -c "True")" -eq 1 ];
+  echo "Waiting for kube-apiserver-operator"
+  until [ "$(oc get pod -n openshift-kube-apiserver-operator --selector='app=kube-apiserver-operator' -o jsonpath='{.items[*].status.conditions[?(@.type=="Ready")].status}' | grep -c "True")" -eq 1 ];
```

Comment:
The 55a53adb910e2 commit message says:

> Use [*] and not [0] in the jsonpath

But doesn't explain the benefit of that pivot.  If there  happen to be multiple operator pods,isn't that ok?  I guess we only get one line out of your new JSON Path,  and the logic is moving from "is the randomly-sorted-first pod ready?" to "are there any ready operator pods?"?

Reply:
I made this change because if the pod doesn't exist yet we get a nasty output.
I updated the commit message.